ts/main.ts

```diff
@@ -127,3 +155,25 @@ $divEntries.addEventListener('click', (event: Event): void => {
     viewSwap('entry-form');
   }
 });
+
+$ul.addEventListener('click', (event: Event): void => {
+  const eventTarget = event.target as HTMLElement;
+  if (eventTarget.matches('.fa-pencil')) {
+    const $li = eventTarget.closest('li');
+    const entryId = $li?.getAttribute('data-entry-id');
+    data.editing =
+      data.entries[Math.abs(Number(entryId) - data.entries.length)];
```

Comment:
This won't always work. Eventually we're going to be able to delete entries, and the id's won't reset. You need to find another way to find the element inside of `data.entries`